examples/tutorials/contour-map.py

```diff
@@ -0,0 +1,103 @@
+"""
+Creating a map with contour lines
+=================================
+
+Plotting a contour map is handled by :meth:`pygmt.Figure.grdcontour`.
+"""
+
+import pygmt
+
+# Load sample earth relief data
+grid = pygmt.datasets.load_earth_relief(resolution="15s", region=[-92, -90, -1.5, 0.5])
+
+########################################################################################
+# Create contour plot
+# -------------------
+#
+# The :meth:`pygmt.Figure.grdcontour` method takes the grid input.
+# It plots annotated contour lines, which are darker and have the
```

Comment:
The default `pen` for annotated and regular contours are `0.75p,black` and `0.25p,black`, respectively. It's not exactly correct saying that `0.75p,black` is "darker" than `0.25p,black`. So I suggest changing "darker" to "thicker".
```suggestion
# It plots annotated contour lines, which are thicker and have the
```